arches/app/datatypes/datatypes.py

```diff
@@ -1374,18 +1374,50 @@ def get_compatible_renderers(self, file_data):
 
     def transform_value_for_tile(self, value, **kwargs):
         """
-        Accepts a comma delimited string of file paths as 'value' to create a file datatype value
-        with corresponding file record in the files table for each path. Only the basename of each path is used, so
-        the accuracy of the full path is not important. However the name of each file must match the name of a file in
-        the directory from which Arches will request files. By default, this is the directory in a project as defined
-        in settings.UPLOADED_FILES_DIR.
-
+        Accepts a comma delimited string of file paths as 'value'
+        or a dictionary / a list of dictionaries as following
```

Comment:
I think I might have originally written this docstring. Something closer to this would have been clearer:
```suggestion
        The 'value' argument can be a comma delimited string of file paths, a dictionary, or a list of dictionaries with the following properties:
```